cranelift/filetests/filetests/isa/x64/i128.clif

```diff
@@ -1701,12 +1705,14 @@ block0(v0: i128, v1: i128):
 ;   movq %rsp, %rbp
 ; block1: ; offset 0x4
 ;   movq %rdx, %rcx
+;   movq %rdx, %r11
 ;   movq %rdi, %r8
 ;   shrq %cl, %r8
 ;   movq %rsi, %r10
 ;   shrq %cl, %r10
+;   movq %rcx, %r11
 ;   movl $0x40, %ecx
-;   movq %rdx, %rdi
+;   movq %r11, %rdi
```

Comment:
The additional moves here are expected, as we're now creating overlapping liveranges in regalloc2-0.9.0. Follow-on work will involve removing these, but local profiling of benchmarks like bz2, pulldown-cmark, and spidermonkey was not showing any performance regression.